src/module/hooks/handlebars.js

```diff
@@ -245,6 +245,16 @@ function registerHandlebarsHelpers() {
 	Handlebars.registerHelper("isMonsterTypeMount", function (type) {
 		return type === "mount";
 	});
+
+	Handlebars.registerHelper("chargenLoc", function (item) {
+		console.log(item);
+		let localizedString = game.fbl.config.attributes[item];
+		if (!localizedString) {
+			const SKILL_NAME = item.toUpperCase().replace(/\s/g, "_");
+			localizedString = `SKILL.${SKILL_NAME}`;
+		}
+		return localizedString;
+	});
```

Comment:
Moved translation into Handlebars instead. The previously noted issue with attributes/skills not translating correctly when changing the childhood is now resolved.